### PR TITLE
Added 2x2 citaion view template

### PR DESCRIPTION
**Pull request by AlexeyGreer**
Screen Shot 2023-03-01 at 3 18 42 PM

### CODE REVIEW

File: src/foam/u2/TwoColumnCitationView.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/**
```
Comment (sarthak-marwaha) [inline src/foam/u2/TwoColumnCitationView.js:1]: Foam header

File: src/foam/u2/TwoColumnCitationView.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      font-size: 1rem;
+    }
+    ^row-1-section-1{
+
+    }
+    ^row-1-section-2{
+      display: flex;
+      gap: 0.5rem;
+    }
+    ^row-2-section-1{
```
Comment (sarthak-marwaha) [inline src/foam/u2/TwoColumnCitationView.js]: remove empty css classes

File: src/foam/u2/TwoColumnCitationView.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    }
+    ^primary{
+      font-weight: bold;
+      color: $black;
+    }
+  `,
+
+  methods: [
+    //Template
```
Comment (sarthak-marwaha) [inline src/foam/u2/TwoColumnCitationView.js]: remove comment

File: src/foam/u2/TwoColumnCitationView.js (hunk 1 above)
Comment (AlexeyGreer) [reply src/foam/u2/TwoColumnCitationView.js:1]: This will shrink my line count tho

**Pull request merged**